tests/client/service/nodesAdd.test.ts

```diff
@@ -128,16 +130,17 @@ describe('nodesAdd', () => {
     });
   });
   afterEach(async () => {
+    await taskManager.startProcessing();
```

Comment:
This should be `stopProcessing()`. Check all the rest as well.

Reply:
This has been checked for all tests and fixed.